resources/repo.rb

```diff
@@ -16,7 +16,9 @@ def apiclient
   api_password desired.api_password
 
   begin
-    config = JSON.parse(apiclient.run_script('get_repo', desired.repo_name))
+    res = apiclient.run_script('get_repo', desired.repo_name)
+    current_value_does_not_exist! if res == 'null'
+    config = JSON.parse(res)
     current_value_does_not_exist! if config.nil?
```

Comment:
I'm ok you add the `if res == 'null'` test, but if you do so please remove the `config.nil?` check